product-features/self-service-listing.md

```diff
@@ -132,15 +132,29 @@ Recommended to track transaction status:
 
 </details>
 
-### Step 2: Contract creation
+### Step 2: LP Lock & Burn Settings
+
+After submitting the Self-Service Listing Pool, a pop-up will appear, allowing the creator to lock or burn the LP tokens, or to leave the liquidity pool unlocked. By default, the Self-Service Listing Pool is set to be locked for 6 months, as it is the recommended option. Users are prompted to select one of three settings:
+
+- **Do not lock LP 🔓**: There will be no lock-up period and the initial liquidity provider will receive the corresponding LP tokens once the pool is live and operational. Since the pool is unlocked, the owner will be able to withdraw liquidity at any time. 
```

Comment:
There will be no lock-up period and the initial liquidity provider _(the pool creator)_ will receive the corresponding LP tokens once the pool is live and operational.